docs/quick-start/index.md

```diff
@@ -168,7 +168,7 @@ We are not reviewing these parts of the project configuration deeper because the
 related to the use of the Spine framework. If you're interested in more details, please look into
 the code of these scripts.
 
-Now, let's look into the definition of the data structures of the Hello context. 
+Now, let's look into the data structures of the Hello context. 
```

Comment:
Maybe `structure` would read better? There is also `The data of the Hello context are defined ...` down below. 

Both `structures` and `the data are defined` seem to be examples of a way-too-literal translation from Russian.